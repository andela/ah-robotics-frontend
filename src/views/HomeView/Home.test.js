import React from 'react';
import HomeView from '.';

it('app should render Home page', () => {
  const wrapper = shallow(<HomeView />);
  expect(wrapper.length).toBe(1);
});
it('app should match the snapshop', () => {
  const wrapper = shallow(<HomeView />);
    expect(wrapper).toMatchSnapshot();
});
