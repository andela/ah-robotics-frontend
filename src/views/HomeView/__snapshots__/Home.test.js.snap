// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should match the snapshop 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
